lib/twingly/public_suffix_list.rb

```diff
@@ -1,3 +1,4 @@
+require "addressable/idna"
```

Comment:
This was missing from before, and the specs will fail without it (because the spec runs this file directly – it was required indirectly before)